cli/lsp/analysis.rs

```diff
@@ -456,11 +456,15 @@ impl CodeActionCollection {
   ) -> Result<(), AnyError> {
     if let Some(data) = diagnostic.data.clone() {
       let fix_data: DenoFixData = serde_json::from_value(data)?;
+      let title = if diagnostic.code
+        == Some(lsp::NumberOrString::String("no-cache-data".to_string()))
```

Comment:
If you want to avoid such `to_string()` calls, you can use `matches!`:
```suggestion
      let title = if matches!(diagnostic.code, Some(lsp::NumberOrString::String(code)) if code == "no-cache-data")
```
(Or just use `match`, of course.)

It's not that a single allocation is terrible but it's often death by a thousand cuts.